src/applications/personalization/dashboard-2/tests/e2e/loa3.cypress.spec.js

```diff
@@ -45,7 +45,6 @@ function loa3DashboardTest(mobile) {
 
   // focus should be on the h1
   cy.focused()
-    .should('have.attr', 'id', 'dashboard-title')
```

Comment:
Kind of a dumb assertion to make here. I can say that because I put it there in the first place.

Reply:
😆